### PR TITLE
Add Keypad Layout 1.1

### PR DESCRIPTION
**Pull request by janten**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **updating a cask**:

- [ ] [If the `sha256` changed but the `version` didn’t][version-checksum],  
      provide public confirmation by the developer: {{link}}

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/keypad-layout.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'keypad-layout' do
+  version '1.1'
+  sha256 '08c38f7f53a4112254b6504b8e91aa422b507f646df7f8281df56b8721cb958c'
+
+  url "https://github.com/janten/keypad-layout/releases/download/#{version}/Keypad-Layout.zip"
+  name 'Keypad Layout'
```
Comment (vitorgalvao) [inline Casks/keypad-layout.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/keypad-layout.rb (hunk 1 above)
Comment (janten) [inline Casks/keypad-layout.rb]: Appcast has been added.

**Comment by janten** [comment]
@vitorgalvao I have added the requested appcast and amended the original commit.

**Review by janten — COMMENTED** [review]
Suggested changes have been incorporated into the commit.

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**